n=35
0: symptoms/asymptomatic
1: symptoms/headache1
2: symptoms/headache2
3: symptoms/headache3
4: symptoms/headache4
5: symptoms/headache_bool
6: symptoms/nohup
7: symptoms/symptom0
8: symptoms/symptom10
9: symptoms/symptom11
10: symptoms/symptom12
11: symptoms/symptom13
12: symptoms/symptom14
13: symptoms/symptom15
14: symptoms/symptom16
15: symptoms/symptom17
16: symptoms/symptom18
17: symptoms/symptom19
18: symptoms/symptom1
19: symptoms/symptom20
20: symptoms/symptom21
21: symptoms/symptom22
22: symptoms/symptom23
23: symptoms/symptom24
24: symptoms/symptom25
25: symptoms/symptom26
26: symptoms/symptom27
27: symptoms/symptom2
28: symptoms/symptom3
29: symptoms/symptom4
30: symptoms/symptom5
31: symptoms/symptom6
32: symptoms/symptom7
33: symptoms/symptom8
34: symptoms/symptom9

i=0
running=0
Run symptoms/asymptomatic

i=1
running=1
Run symptoms/headache1
./run_parallel.sh: line 48:  8965 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=2
running=1
Run symptoms/headache2
./run_parallel.sh: line 48:  8978 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=3
running=1
Run symptoms/headache3
./run_parallel.sh: line 48:  8993 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=4
running=1
Run symptoms/headache4
./run_parallel.sh: line 48:  9006 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=5
running=1
Run symptoms/headache_bool
./run_parallel.sh: line 48:  9019 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=6
running=1
Run symptoms/nohup
./run_parallel.sh: line 48:  9032 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=7
running=1
Run symptoms/symptom0
Traceback (most recent call last):
  File "/home/oevans/chiari/python/learn.py", line 159, in <module>
    alldata=loadtxt('data/{}.txt'.format(data_name))
  File "/home/oevans/local/miniconda3/lib/python3.5/site-packages/numpy/lib/npyio.py", line 803, in loadtxt
    fh = iter(open(fname))
FileNotFoundError: [Errno 2] No such file or directory: 'data/symptoms/nohup.txt'
Using output name: 'symptoms/nohup-04-06-2016'
Opening file for logging: logs/symptoms/nohup-04-06-2016.log
Current script name: learn
Data name: symptoms/nohup

i=8
running=1
Run symptoms/symptom10
./run_parallel.sh: line 48:  9059 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=9
running=1
Run symptoms/symptom11
./run_parallel.sh: line 48:  9071 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=10
running=1
Run symptoms/symptom12
./run_parallel.sh: line 48:  9085 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=11
running=1
Run symptoms/symptom13
./run_parallel.sh: line 48:  9098 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=12
running=1
Run symptoms/symptom14
./run_parallel.sh: line 48:  9111 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=13
running=1
Run symptoms/symptom15
./run_parallel.sh: line 48:  9124 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=14
running=1
Run symptoms/symptom16
./run_parallel.sh: line 48:  9137 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=15
running=1
Run symptoms/symptom17

i=16
running=3
./run_parallel.sh: line 48:  9150 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}
./run_parallel.sh: line 48:  9166 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=16
running=0
Run symptoms/symptom18

i=17
running=1
Run symptoms/symptom19
./run_parallel.sh: line 48:  9310 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=18
running=2
Run symptoms/symptom1
./run_parallel.sh: line 48:  9322 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=19
running=2
Run symptoms/symptom20

i=20
running=3
./run_parallel.sh: line 48:  9342 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}
./run_parallel.sh: line 48:  9364 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=20
running=0
Run symptoms/symptom21

i=21
running=1
Run symptoms/symptom22
./run_parallel.sh: line 48:  9495 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=22
running=1
Run symptoms/symptom23
./run_parallel.sh: line 48:  9507 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=23
running=1
Run symptoms/symptom24
./run_parallel.sh: line 48:  9520 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=24
running=1
Run symptoms/symptom25
./run_parallel.sh: line 48:  9533 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=25
running=1
Run symptoms/symptom26
./run_parallel.sh: line 48:  9546 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=26
running=1
Run symptoms/symptom27
./run_parallel.sh: line 48:  9559 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=27
running=1
Run symptoms/symptom2
./run_parallel.sh: line 48:  9572 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=28
running=1
Run symptoms/symptom3
./run_parallel.sh: line 48:  9585 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=29
running=1
Run symptoms/symptom4
./run_parallel.sh: line 48:  9598 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=30
running=1
Run symptoms/symptom5
Traceback (most recent call last):
  File "/home/oevans/chiari/python/learn.py", line 343, in <module>
    fit_algorithm(alg_num)
  File "/home/oevans/chiari/python/learn.py", line 95, in fit_algorithm
    learner.fit(train_data[:,feature_numbers],train_data[:,-1])
  File "/home/oevans/local/miniconda3/lib/python3.5/site-packages/sklearn/svm/base.py", line 151, in fit
    y = self._validate_targets(y)
  File "/home/oevans/local/miniconda3/lib/python3.5/site-packages/sklearn/svm/base.py", line 521, in _validate_targets
    % len(cls))
ValueError: The number of classes has to be greater than one; got 1
Using output name: 'symptoms/symptom4-04-06-2016'
Opening file for logging: logs/symptoms/symptom4-04-06-2016.log
Current script name: learn
Data name: symptoms/symptom4
n_samples = 99
n_features = 15
1 positives
98 negatives
7 algorithms: ['SVM', '4NN', 'DT', 'LDA', 'QDA', 'NB', 'LR']
6 sizes: [2, 3, 4, 5, 6, 7]
Total # of tuples: 16368

Tuple size: 2
n_tuples = 105

i=31
running=1
Run symptoms/symptom6
./run_parallel.sh: line 48:  9624 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=32
running=1
Run symptoms/symptom7
./run_parallel.sh: line 48:  9636 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=33
running=1
Run symptoms/symptom8
./run_parallel.sh: line 48:  9649 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}

i=34
running=1
Run symptoms/symptom9
./run_parallel.sh: line 48:  9662 Illegal instruction     nohup python "/home/oevans/chiari/python/learn.py" ${args[i]}
